### DISCUSSION TITLE
Nachrichten: Ausblenden von Konversationen

### DISCUSSION THREAD

**Discussion by techotline** [lotta-schule/web#241] [Ideen]
### Feature

Das Löschen einer Konversation  und der alle damit verbundenen Nachrichten selbst, die mit diesem Nutzer ausgetauscht wurden, sollte möglich gemacht werden.

### Idee

Es könnte sich Dialog öffnen, der A:

* den Nutzer (Empfänger; Person B) über die Löschung mithilfe einer Ablauffrist zur Möglichkeit über die Sicherung des Gesprächsprotokolls von bspw. 14 Tagen informiert
* oder B) die Nachrichten + Konversation nur noch einerseits entfernt werden und die Nachrichten bei dem Sender; Person A bzw. Empfänger; Person B erhalten bleiben können, sofern Person A oder B den Vorgang ausführt und Nutzer A / B die Nachrichten für Nachweiszwecke aufbewahren möchte.

Grundsätzlich sollte man sowohl die Entscheidung treffen als auch die Kontrolle behalten können, wann und ob man Nachrichten löscht.

**Comment by ptitmouton** [comment]
Warum?

**Comment by ptitmouton** [comment]
Lösung: ausblenden
